Solutions/Dynatrace/Playbooks/Add_DynatraceApplicationSecurityAttackSourceIpThreatIntelligence/readme.md

```diff
@@ -0,0 +1,46 @@
+author: Dynatrace
+
+This playbook uses the Dynatrace REST APIs to automatically add an attackers source IP address to Microsoft Threat Intelligence when an alert is generated by Microsoft Sentinel. You need a valid Dynatrace Tenant and Access Token, the token should have Read attacks (attacks.read) scope in order to use the playbook. To learn more about the Dynatrace platform [Start your free trial](https://www.dynatrace.com/trial)
+
+** Prerequisites ** 
+- Follow [these instructions](https://www.dynatrace.com/support/help/get-started/access-tokens#create-api-token) to generate a Dynatrace access token.
+- [Important step]Store the Dynatrace Access Token as a secret in Azure Key vault and provide the key vault name during deployment, by convention the secret name should be 'DynatraceAccessToken'.
+
+** Post Install Notes:**
+
+The Logic App creates and uses a Managed System Identity (MSI) to update the Microsoft Sentinel Incident. 
+
+Assign RBAC 'Microsoft Sentinel Reader' role to the Logic App at the Resource Group level of the Log Analytics Workspace.
+
+Assign access policy on key vault for Playbook to fetch the secret key
+
+To create new Threat Indicators for Microsoft Threat Intelligence we leverage the Microsoft Graph Security connector, please ensure you meet the prerequisites to connect with The [Microsoft Graph Security connector](https://learn.microsoft.com/en-us/connectors/microsoftgraphsecurity/#prerequisites-to-connect-with-the-microsoft-graph-security-connector)
+
+Basic steps for the Microsoft Graph Security connector permissions are as follows :
+
+1. Perform role assignment for the playbook managed indentity, assign the ThreatIndicators.ReadWrite.OwnedBy permission  
+
+    # Install the AzureAD PowerShell module
+    # Install-Module AzureAD
+    $TenantID="[YOUR AZURE ACTIVE DIRECTORY TENANT ID]"
+    $GraphAppId = "00000003-0000-0000-c000-000000000000"
+    $DisplayNameOfMSI="[PLAYBOOK IDENTITY NAME]"
+    $PermissionName = "ThreatIndicators.ReadWrite.OwnedBy"
+    Connect-AzureAD -TenantId $TenantID
+    $MSI = (Get-AzureADServicePrincipal -Filter "displayName eq '$DisplayNameOfMSI'")
+    Start-Sleep -Seconds 10
+    $GraphServicePrincipal = Get-AzureADServicePrincipal -Filter "appId eq '$GraphAppId'"
+    $AppRole = $GraphServicePrincipal.AppRoles | Where-Object {$_.Value -eq $PermissionName -and $_.AllowedMemberTypes -contains "Application"}
+    New-AzureAdServiceAppRoleAssignment -ObjectId $MSI.ObjectId -PrincipalId $MSI.ObjectId -ResourceId $GraphServicePrincipal.ObjectId -Id $AppRole.Id
+
+2. Grant Admin Consent for MicrosoftGraphSecurityConnector application
+2.1 Navigate to 'https://login.microsoftonline.com/[YOUR AZURE ACTIVE DIRECTORY TENANT ID]/adminconsent?client_id=c4829704-0edc-4c3d-a347-7c4a67586f3c'
+2.2 Accept the requested permissions
+
+
+## Initial Setup
+
+Logic App is utilitzed by Automation rules. You must setup an Automation rule. Go to the Automation Rules blade in Azure Sentinel. 
+
```

Comment:
Please elaborate more , on how customer or user can attach this playbook with automation rule , alert or incident .
Give the steps by step

Reply:
@manishkumar1991 - I have added additional steps